### PR TITLE
Named schedule

### PR DESCRIPTION
**Pull request by klydel**
Add a string name to a scheduled task for easier viewing via Entries.

### CODE REVIEW

**Comment by klydel** [comment]
Sorry about the mess.  Feel free to let me know if ive done this wrong.

File: cron.go [hunk 1]
```diff
@@ -83,25 +85,26 @@ type FuncJob func()
...
 // AddFunc adds a func to the Cron to be run on the given schedule.
-func (c *Cron) AddFunc(spec string, cmd func()) error {
-	return c.AddJob(spec, FuncJob(cmd))
+func (c *Cron) AddFunc(spec string, cmd func(), n string) error {
+	return c.AddJob(spec, FuncJob(cmd), n)
 }
 
 // AddFunc adds a Job to the Cron to be run on the given schedule.
-func (c *Cron) AddJob(spec string, cmd Job) error {
+func (c *Cron) AddJob(spec string, cmd Job, n string) error {
```
Comment (shurcooL) [inline cron.go]: `n string` is not a very descriptive parameter. Someone seeing this library for the first time will likely not immediately understand what `n` represents. `name` would be better.

Also, the documentation currently says nothing about the name parameter, perhaps it should say "name is used for viewing purposes only".

Comment (klydel) [inline cron.go]: Good catch, thanks for the feedback, i will change this.

**Comment by robfig** [comment]
Job is an interface, so you can create your own type that implements String() and has whatever properties or printing that you desire. That is easy to do, so I don't think it's worth baking it in and breaking backwards compatibility.
Sorry for the delay in responding!

**Pull request closed without merging**